code/modules/clothing/spacesuits/rig/modules/modules.dm

```diff
@@ -175,10 +175,6 @@
 	if(!holder.check_power_cost(usr, use_power_cost, 0, src, (istype(usr,/mob/living/silicon ? 1 : 0) ) ) )
 		return 0
 
-	if(!confined_use && istype(usr.loc, /obj/mecha))
```

Comment:
This check must stay. You just need to replace the /obj/mecha with the mech mob path.